explore/src/main/scala/explore/config/SequenceTable.scala

```diff
@@ -89,142 +89,144 @@ object SequenceTable {
   private val offsetFormat = new DecimalFormat("#.0")
 
   private val StepTable = TableMaker[StepLine[_]]
+  import StepTable.syntax._
 
-  private val StepTableComponent = new SUITable(StepTable)
+  private def drawBracket(rows: Int): VdomElement =
+    svg(^.width := "1px", ^.height := "15px", ^.overflow.visible)(
+      use(
+        transform := s"scale(1, $rows)",
+        xlinkHref := "#bracket"
+      )
+    )
+
+  private def rightAligned(value: Any) =
+    <.div(^.textAlign.right)(value.toString)
+
+  private val columns = List(
+    StepTable
+      .Column("atomSteps", _.firstOf)
+      .setHeader(" ")
+      .setCell(_.value.map(drawBracket)),
+    StepTable
+      .Column("stepType", _.step.stepType)
+      .setHeader("Type")
+      .setCell(_.value.toString),
+    StepTable
+      .Column("exposure", _.exposureSecs)
+      .setHeader(rightAligned("Exp (sec)").rawElement)
+      .setCell(cell => rightAligned(cell.value)),
+    StepTable
+      .Column("guide", _.guided)
+      .setHeader("")
+      .setCell(cell =>
+        if (cell.value) Icons.Crosshairs.copy(clazz = ExploreStyles.StepGuided) else EmptyVdom
+      ),
+    StepTable
+      .Column("p", _.p)
+      .setHeader(rightAligned("p").rawElement)
+      .setCell(cell => rightAligned(offsetFormat.format(cell.value))),
+    StepTable
+      .Column("q", _.q)
+      .setHeader(rightAligned("q").rawElement)
+      .setCell(cell => rightAligned(offsetFormat.format(cell.value))),
+    StepTable
+      .Column("lambda", _.wavelength)
+      .setHeader(rightAligned("λ (nm)").rawElement)
+      .setCell(_.value.map((rightAligned _).compose(_.toInt))),
+    StepTable
+      .Column("fpu", _.fpuName)
+      .setHeader(rightAligned("FPU").rawElement)
+      .setCell(_.value.map(rightAligned)),
+    StepTable
+      .Column("disperser", _.disperserName)
+      .setHeader("Disperser")
+      .setCell(_.value.orEmpty),
+    StepTable
+      .Column("filter", _.filterName)
+      .setHeader("Filter")
+      .setCell(_.value.orEmpty),
+    StepTable
+      .Column("xbin", _.step.instrumentConfig.readout.xBin)
+      .setHeader(rightAligned("Xbin").rawElement)
+      .setCell(cell => rightAligned(cell.value.shortName)),
+    StepTable
+      .Column("ybin", _.step.instrumentConfig.readout.yBin)
+      .setHeader(rightAligned("Ybin").rawElement)
+      .setCell(cell => rightAligned(cell.value.shortName)),
+    StepTable
+      .Column("roi", _.step.instrumentConfig.roi)
+      .setHeader("ROI")
+      .setCell(_.value.shortName),
+    StepTable
+      .Column("sn", _ => "")
+      .setHeader("S/N")
+  )
+
+  implicit private val colReuse: Reusability[List[ColumnInterface[StepLine[_]]]] =
+    Reusability.always
+  implicit private val dataReuse: Reusability[List[StepLine[_]]]                 = Reusability.byRefOr_==
```

Comment:
Does this mean we run reusability for each row? I guess with a virtualized list it is fine but maybe slow for large non virtualized tables?

Reply:
I agree we don't want to check all the elements in the list. That's why it's implemented as `byRefOr_==`.